docs/api-reference/core/deck.md

```diff
@@ -691,6 +691,10 @@ Notes:
 
 ## Member Variables
 
+##### `isLoaded`
+
+Flag indicating that the Deck instance has initialized its resources.
```

Comment:
It may be good add a bit more detail. Because especially with tile layers, data constantly unloads and loads as the user pans.

1. We could mention that this is for load of initial resource load only (if I understand this correctly) 
2. we do have a technique for constantly rechecking if everything is loaded after every interaction (something like iterating over layers and checking if they are loaded), this technique could be mentioned here for completeness.

Reply:
That's not what this is for. The reason we have this is so components can check if the `viewManager` is initialized (as it is now private) and get the viewports. Maybe `isReady` is a better name?

Reply:
Hmm I see. 

Yes then it could make sense to have more specific naming. 

Keep in mind that there are other initializations going on, like WebGL/WebGPU initialization.  Should we bundle these all together in an `onInitialized` `isInitialized` kind of callback, or do we need these super granular notifications for each initialization category? Maybe a 'ViewManagerInitialized" event?  

But is so does that really need to be a public documented API?